src/bp/core/services/dialog/flow/service.ts

```diff
@@ -13,6 +13,8 @@ const FLOW_DIR = 'flows'
 
 @injectable()
 export class FlowService {
+  private _allFlows: Map<string, FlowView[]> = new Map()
```

Comment:
need to invalidate too on flow saved

Reply:
Already there. check at the end of saveAll ;)